.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "go.buildTags": "all",
+    "go.testTimeout": "1h",
+    "gopls": {
+        // A couple of modules get copied as part of builds and this confuse gopls as it sees the module name twice, just ignore the copy in the build folders.
```

Comment:
Is it okay for this file to be invalid JSON?

Reply:
Yeah, VSCode is happy with comments in settings - this is pretty much a copy from pulumi/pulumi.